utils/eventtracking.js

```diff
@@ -117,8 +127,8 @@ const askForConsentIfNeeded = async (options) => {
 
 const trackDeployedContract = async () => {
     const shellSettings = settings.getShellSettings();
-    const id = getMixpanelID(shellSettings);
-    await mixpanel.people.increment(id, 'deployed_contracts');
+    const id = getSegmentID(shellSettings);
+    // TODO: find a way to increment a Segment analytic property
```

Comment:
It seems like there is no persistent analytic properties in segment (or at least a direct replacement). Not sure how much we care about the total amount of deployed contracts as a stored number vs aggregating them by `deployed` events.

Reply:
@TiffanyGYJ